static/app/views/replays/detail/console/useConsoleFilters.tsx

```diff
@@ -85,8 +85,14 @@ function useConsoleFilters({breadcrumbs}: Options): Return {
     [breadcrumbs]
   );
 
-  const logLevel = decodeList(query.f_c_logLevel).filter(isBreadcrumbTypeValue);
-  const searchTerm = decodeScalar(query.f_c_search, '').toLowerCase();
+  const logLevel = useMemo(
+    () => decodeList(query.f_c_logLevel).filter(isBreadcrumbTypeValue),
+    [query.f_c_logLevel]
+  );
+  const searchTerm = useMemo(
+    () => decodeScalar(query.f_c_search, '').toLowerCase(),
+    [query.f_c_search]
+  );
```

Comment:
`searchTerm` is a scalar, useMemo does something there?

for the `logLevel` array we could probably do the same in the order `useXYZFilters.tsx` files.